### PR TITLE
[12.0][MIG] Base Partner Sequence

### PR DESCRIPTION
**Pull request by qgroulard**
Migration of base_partner_sequence to 12.0

### CODE REVIEW

**Review by gva-acsone — APPROVED** [review]
👍

File: base_partner_sequence/__manifest__.py [hunk 1]
```diff
@@ -5,22 +5,17 @@
...
-              "Odoo Community Association (OCA), "
-              "Camptocamp, ",
+    "ACSONE SA/NV,"
+    "Tecnativa, "
+    "Odoo Community Association (OCA), "
+    "Camptocamp, ",
     "name": "Add a sequence on customers' code",
-    "version": "11.0.1.0.0",
+    "version": "12.0.1.0.0",
     "category": "Generic Modules/Base",
```
Comment (rousseldenis) [inline base_partner_sequence/__manifest__.py:17]: @qgroulard Maybe adding development_status https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L9

**Review by rousseldenis — APPROVED** [review]

File: base_partner_sequence/__manifest__.py (hunk 1 above)
Comment (qgroulard) [reply base_partner_sequence/__manifest__.py:17]: Done.

File: base_partner_sequence/tests/test_base_partner_sequence.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+# Copyright 2015 ACSONE SA/NV (<http://acsone.eu>).
+# Copyright 2016 Tecnativa - Vicent Cubells
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import openerp.tests.common as common
```
Comment (nikul-serpentcs) [inline base_partner_sequence/tests/test_base_partner_sequence.py]: odoo

**Review by nikul-serpentcs — CHANGES_REQUESTED** [review]
Minor Change

File: base_partner_sequence/tests/test_base_partner_sequence.py (hunk 2 above)
Comment (qgroulard) [reply base_partner_sequence/tests/test_base_partner_sequence.py]: Fixed

**Review by nikul-serpentcs — APPROVED** [review]
Code Review LGTM :+1:

**Pull request merged**